pkg/template/apis/template/types.go

```diff
@@ -108,10 +108,24 @@ type TemplateInstanceSpec struct {
 // TemplateInstanceRequester holds the identity of an agent requesting a
 // template instantiation.
 type TemplateInstanceRequester struct {
-	// Username is the username of the agent requesting a template instantiation.
+	// The name that uniquely identifies this user among all active users.
```

Comment:
sorry, it's the comments that I'm referring to which should start with lower case.  Of course the field names must be capitalised.